### PR TITLE
Blas instructions

### PR DESCRIPTION
**Pull request by isuruf**
Last PR was from a branch. Resubmitting from a fork

### CODE REVIEW

File: src/maintainer/knowledge_base.rst [hunk 1]
```diff
@@ -554,45 +554,96 @@ Special packages
...
-   -  build number cannot be touched (it won't be in the string anyways)
-   -  no pinning of BLAS inside the metapackages (dependencies can pin
-      this)
-   -  to preserve a BLAS in an environment it is recommend to add
-      ``pinned`` to ``conda-meta`` and specify down to the build string
-      what is the expected BLAS
-   -  To install a specific variant, ``conda install blas=1.0=none`` /
-      ``conda install blas=1.0=openblas``. It is hoped the syntax will be
-      improved in conda.
+``libblas`` and `libcblas`` versioning is based on the Reference LAPACK versioning which at the
```
Comment (loriab) [inline src/maintainer/knowledge_base.rst]: missed a backtick for `libcblas` here and two lines hence

File: src/maintainer/knowledge_base.rst [hunk 2]
```diff
@@ -554,45 +554,96 @@ Special packages
...
-      -  Y represents priority of BLAS (if we change priorities BLASes X
-         must be bumped, if we want to prioritize something new over
-         OpenBLAS we do not need to change X)
+How it works
+""""""""""""
 
-         -  1 is OpenBLAS
-         -  0 is None (no BLAS whatsoever)
+At recipe build time, the netlib packages are used. This means that the downstream package will
+link to ``libblas.so.3`` in the ``libblis=*=*netlib` and will use only the reference
```
Comment (loriab) [inline src/maintainer/knowledge_base.rst]: "blis" is on the lhs of the `{package}={version}={buildstr}` here and in the paragraph around line 640; whereas, I'm accustomed to seeing it in the buildstr slot. Typo or something I should learn about?

File: src/maintainer/knowledge_base.rst (hunk 1 above)
Comment (isuruf) [reply src/maintainer/knowledge_base.rst]: Thanks

File: src/maintainer/knowledge_base.rst (hunk 2 above)
Comment (isuruf) [reply src/maintainer/knowledge_base.rst]: Should have been `libblas` instead of `libblis`. Thanks

**Comment by scopatz** [comment]
Is this good to go?

**Comment by isuruf** [comment]
Yes

**Pull request merged**